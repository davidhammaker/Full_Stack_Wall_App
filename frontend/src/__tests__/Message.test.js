import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Message from '../components/message';


configure({ adapter: new Adapter() });


it('displays message correctly', () => {
  const message = renderer.create(
    <Message message='message' />
  );
  let tree = message.toJSON();
  expect(tree).toMatchSnapshot();
});

it('displays nothing if no message', () => {
  const message = shallow(<Message />);
  expect(message.contains(<></>)).toBe(true);
});
