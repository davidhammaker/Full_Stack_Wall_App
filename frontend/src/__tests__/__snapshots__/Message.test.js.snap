// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays message correctly 1`] = `
<small>
  <div
    className="alert alert-success"
  >
    message
  </div>
</small>
`;
